app/src/main/res/layout/item_collectible_collectible.xml

```diff
@@ -12,6 +12,7 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_gravity="center"
+        android:layout_marginTop="2dp"
```

Comment:
This only increases the distance between the items, right? It does not add any more shadow.

Reply:
top shadow/line was clipped by view bounds. We can fix it by either adding padding to parent or having this margin